example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,157 @@
+
+# Check if .env file is present
+if [ ! -f ".env" ]; then
+  echo "Error: '.env' file not found. Exiting." | tee -a "$LOG_FILE"
+  exit 1
+fi
+echo ".env file found in $pwd directory. Proceeding..." | tee -a "$LOG_FILE"
+
+# Load environment variables from .env file securely
+while IFS= read -r line || [ -n "$line" ]; do
+  if [[ ! "$line" =~ ^# && "$line" =~ = ]]; then
+    export "$line"
```

Shellcheck (0.10.0)

[warning] This does not export 'line'. Remove $/${} for that, or use ${var?} to quiet.

(SC2163)